eclipsePlugin/build.gradle

```diff
@@ -74,6 +74,7 @@ task copyLibsForEclipse (type: Copy) {
     into "lib"
     include "*.jar"
     exclude "*xml-apis*.jar"
+    exclude "jdepend*.jar"
```

Comment:
FYI, I'll be tackling this and some other minor gradle improvements. I'll submit a PR after I test it.